doi/src/main/webapp/paths/doi-instance-search.yaml

```diff
@@ -0,0 +1,20 @@
+post:
+  tags:
+    - "instances"
+  summary: "Get a list of DOI instances"
+  description: "Get DOI instances based on filter request."
+  operationId: "getDOIs"
+  parameters:
+    - $ref: "../components/parameters/status.yaml#/status"
+    - $ref: "../components/parameters/role.yaml#/role"
```

Comment:
Do these refs render correctly for you when viewing openapi.yaml? I see search parameters with no name or values. Could just be my IDE.